src/nvim/ui.c

```diff
@@ -235,6 +235,36 @@ int ui_pum_get_height(void)
   return pum_height;
 }
 
+void ui_pum_get_pos(double *pwidth, double *pheight, double *prow, double *pcol)
+{
+  double w = 0.0, h = 0.0, r = 0.0, c = 0.0;
+  bool found = false;
+  for (size_t i = 1; i < ui_count; i++) {
+    if (!uis[i]->pum_pos) {
+      continue;
+    }
+    w = uis[i]->pum_width;
```

Comment:
you can assign `pwidth` etc directly here and call `return`. then `found` var can be eliminated.